File: v2/csharp/src/main/csharp/io/swagger/Model/Newslettersettingresponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Newslettersettingresponse {
    

    
    public NewsletterSetting NewsletterSetting { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Newslettersettingresponse {\n");
      
      sb.Append("  NewsletterSetting: ").Append(NewsletterSetting).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/android/src/main/java/io/swagger/client/model/Newslettersettingresponse.java
package io.swagger.client.model;

import io.swagger.client.model.NewsletterSetting;

import com.wordnik.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class Newslettersettingresponse  {
  
  @SerializedName("newsletter_setting")
  private NewsletterSetting newsletterSetting = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  public NewsletterSetting getNewsletterSetting() {
    return newsletterSetting;
  }
  public void setNewsletterSetting(NewsletterSetting newsletterSetting) {
    this.newsletterSetting = newsletterSetting;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Newslettersettingresponse {\n");
    
    sb.append("  newsletterSetting: ").append(newsletterSetting).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
